pkg/controller/certificate-shim/helper_test.go

```diff
@@ -31,6 +31,7 @@ import (
 
 func Test_translateAnnotations(t *testing.T) {
 	type testCase struct {
+		name          string
```

Comment:
note: as discussed in standup, I think I understand that this file probably _doesn't_ need to change in this PR, right?